allauth/headless/mfa/response.py

```diff
@@ -38,11 +38,12 @@ class AuthenticatorDeletedResponse(APIResponse):
 
 
 class TOTPNotFoundResponse(APIResponse):
-    def __init__(self, request, secret):
+    def __init__(self, request, secret, totp_url):
         super().__init__(
             request,
             meta={
                 "secret": secret,
+                "totp_url": totp_url,
```

Comment:
This needs to be documented over here: 

https://docs.allauth.org/en/latest/headless/openapi-specification/#tag/Account:-2FA/paths/~1_allauth~1%7Bclient%7D~1v1~1account~1authenticators~1totp/get